components/authz-service/storage/postgres/migration/sql/42_remove_leftover_functions.up.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+
+-- These are leftover, deprecated functions, whose signatures have changed.
```

Comment:
(That is, these should have been included in earlier migrations but were missed.)

Reply:
oh THAT's why there were two of them when we were looking the other day-- makes sense.

Reply:
Yep; decided to track that down.